notebooks/128-openvino-tokenizers/128-opnevino-tokenizers.ipynb

```diff
@@ -0,0 +1,507 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        !pip install -Uq optimum[openvino,openvino-tokenizers,nncf]
</pre><p>there is not release of optimu-intel, where tokenizers support already merged. Also releases of optimum happens on demand, but some times we want to demonstrate the latest features and fixes. In all other notebooks we install optimum-intel from git and I prefer to have it aligned here as well</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1697/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
BTW, optimum-intel just released, but it is still better to use it from git